src/Uno.UI/UI/Xaml/FrameworkElement.Layout.wasm.cs

```diff
@@ -202,11 +202,12 @@ private void InnerArrangeCore(Rect finalRect)
 			var oldRenderSize = RenderSize;
 			var innerInkSize = ArrangeOverride(arrangeSize);
 
-			RenderSize = innerInkSize;
-
 			var clippedInkSize = innerInkSize.AtMost(maxSize);
 
-			_logDebug?.Debug($"{DepthIndentation}{this}: ArrangeResult={innerInkSize}, clipped={clippedInkSize} (max={maxSize}) needsClipToSlot={needsClipToSlot}");
+			//RenderSize = needsClipToSlot ? clippedInkSize : innerInkSize;
```

Comment:
Commented code here does not bring value ?

Reply:
Still valid